packages/amplify-category-auth/src/__tests__/provider-utils/awscloudformation/auth-stack-builder/auth-cognito-stack-builder.test.ts

```diff
@@ -9,9 +9,12 @@ import { CognitoStackOptions } from '../../../../provider-utils/awscloudformatio
 
 describe('generateCognitoStackResources', () => {
   const props: CognitoStackOptions = {
+    // eslint-disable-next-line spellcheck/spell-checker
```

Comment:
1. if possible we should scope it at function level (or perhaps entire file).
2. can you try adding `skipIfMatch` rules that detect hashes? see https://gist.github.com/jrencz/becd1befac3f14b53039e0361087794c#file-eslintrc-L295 as an example.

Reply:
I didn't find any specific rule that matches all those uuid() + resource names.

I think this is better , enabling at top will hinder any future iterations on this file.